react-main/src/redux/login/actions.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import loginService from "../../services/loginService";
+import { push } from 'react-router-redux';
+
+export const actions = {
+  GET_TOKEN_SUCCESS: 'GET_TOKEN_SUCCESS',
+  GET_TOKEN_FAILURE: 'GET_TOKEN_FAILURE'
+};
+
+const actionCreators = {
+  login: (credentials) => async dispatch => {
+    const response = await loginService.login(credentials);
```

Comment:
I commented here before that dispatching an action to "start" the login is useful:

`dispatch({ type: actions.GET_TOKEN });`